doc/lsst.daf.butler/queries.rst

```diff
@@ -352,11 +352,17 @@ Function call
 Function call syntax is similar to other languages, expression for call
 consists of an identifier followed by zero or more comma-separated arguments
 enclosed in parentheses (e.g. ``func(1, 2, 3)``). An argument to a function
-can be any expression.
+can be any expression. Names of the functions are not case-sensitive.
 
-Presently there only one construct that uses this syntax, ``POINT(ra, dec)``
-is function which declares (or returns) sky coordinates similarly to ADQL
-syntax. Name of the ``POINT`` function is not case-sensitive.
+Presently there only few functions implemented in query language:
```

Comment:
```suggestion
Presently the following functions are implemented in the query language:
```
Does make me wonder whether we should add the IVOA `POS` syntax since sphgeom already supports that.